### DISCUSSION TITLE
Multi-label classification using Multi Task / Multi Model

### DISCUSSION THREAD

**Discussion by ghost** [keras-team/autokeras#1450] [Q&A] [answered]
I'm trying to create a model which has N categorical labels (multi-label) and each label has M classes (multi-class), the normal Structured Data Classification works with multi-class and multi-label right? The multi-class works with a specific number or inferring the number of classes, but multi-label its not clear how it works.

Can I use [Multi Task / Multi Model](https://autokeras.com/tutorial/multi/) to achieve multi-label classification? I saw the [nodes](https://autokeras.com/node/), [blocks](https://autokeras.com/block/) and the [example](https://colab.research.google.com/github/keras-team/autokeras/blob/master/docs/ipynb/multi.ipynb#scrollTo=g_Gtg99GUWBM), but I didn't get any hints besides getting the [Sigmoid Cross Entropy With Logits](https://www.tensorflow.org/api_docs/python/tf/nn/sigmoid_cross_entropy_with_logits) and putting it on the search space, which I'm not sure where and if it is possible aside from the nodes / blocks from Auto Keras.

**Comment by haifeng-jin** [comment, marked as answer]
The multi label feature is not like your use case. It is for case like a dataset with M classes and an instance may belong to multiple classes out of M classes.

**Reply by ghost** [reply]
Thanks, I get the idea after finish the prototype that I was working on. I read more on TensorFlow / Keras doc, I switched to Transformers, it makes way more sense for the problem that I need to solve, which has lots of labels and a lot of them are categorical. The Auto ML needed to validate cause the client wants to run it in a specific Cloud that offers it.
